testsuite/ui/views/admin/settings/webhooks.py

```diff
@@ -15,10 +15,10 @@ class WebhooksView(BaseSettingsView):
     path_pattern = '/p/admin/webhooks/edit'
     webhook_active = GenericLocatorWidget('//*[@id="web_hook_active"]')
     webhook_provider = GenericLocatorWidget('//*[@id="web_hook_provider_actions"]')
-    accounts = CheckBoxGroup('//*[@name="Accounts"]')
-    users = CheckBoxGroup('//*[@name="Users"]')
-    applications = CheckBoxGroup('//*[@name="Applications"]')
-    keys = CheckBoxGroup('//*[@name="Keys"]')
+    accounts = CheckBoxGroup('//tr/td', field_set_identifier="Accounts")
```

Comment:
Can't we use here something better than `//tr/td` ? Like `//*[@id="new_web_hook"]/fieldset[@class="inputs"]` Not like this but simmilar